### PR TITLE
Add details about package installation via webUI

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Adding a module describing the new package installation procedure.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
+. Navigate to the *Content* tab > *Packages* subtab.
+. Click the vertical ellipsis icon next to the upgrade button, and select *Install Packages*.
+. A window showing a list of packages available for installation on the host will appear.
+. Select the package or packages that you want to install on the host.
+. Click *Install*.
+
+By default, the packages will be installed using remote execution.
+For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs].
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:16]:
```suggestion
For more information about running remote execution jobs, see {ManagingHostsDocURL}configuring-and-setting-up-remote-jobs_managing-hosts[Configuring and Setting up Remote Jobs] in _{ManagingHostsDocTitle}_.
```

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
+. Navigate to the *Content* tab > *Packages* subtab.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:9]: Unsure about using "navigate" again (which we generally don't) and "subtab" which I've only found two other instances in Foreman documentation. Maybe reword?

**Review by maximiliankolb — COMMENTED** [review]
Some suggestions, but mostly looks good to me! Any thoughts @Lennonka ?

**Comment by AkshayGadhaveRH** [comment]
Hey @maximiliankolb, thanks for your feedback! I'll make these changes. For some reason, I'm not able to select the branches to cherry-pick into or select any of the boxes, in the commit message with either of my browsers. I'll try again with the second commit.

**Comment by AkshayGadhaveRH** [comment]
@maximiliankolb does it make sense to replace it with 'Go to *Content* tab > *Packages*.`? I'm not sure if this seems right.

**Comment by maximiliankolb** [comment]
> @maximiliankolb does it make sense to replace it with 'Go to _Content_ tab > _Packages_.`? I'm not sure if this seems right.

We usually say "... navigate to X > Y". & "On the Z tab, click/select..." Maybe combine the second and third step.

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host.
```
Comment (Lennonka) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:4]: I'd replace this line with something like:
`You can review and install packages on a host using the {WebUI}.` -- sorry, I don't remember the variable name exactly :D

**Review by Lennonka — COMMENTED** [review]
One remark from me, otherwise LGTM!

File: guides/common/modules/proc_installing-packages-on-a-host.adoc (hunk 3 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_installing-packages-on-a-host.adoc:4]: `{ProjectWebUI}`: see https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L84

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 4]
```diff
@@ -0,0 +1,15 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host using the {ProjectWebUI}.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:5]: I'd rather say that it depends on the activation key(s).

the AK depends on a CV, LE, and subscriptions.

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* > select the host you want to install packages on > click *Content* tab.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:8]:
```suggestion
. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
```

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+. On the *Content* tab, click *Packages* > click the vertical ellipsis icon next to the upgrade button, and select *Install Packages*.
+. A window showing a list of packages available for installation on the host will appear.
+. Select the package or packages that you want to install on the host.
+. Click *Install*.
+
+By default, the packages will be installed using remote execution.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:14]:
```suggestion
By default, the packages are installed using remote execution.
```

**Comment by AkshayGadhaveRH** [comment]
Hey @maximiliankolb, ultimately the package profile depends on the repos accessible to the host. So I guess mentioning the repo instead of the subscriptions would be better? I'll check with the SME once again regarding this. 
With regards to Activation Keys, not all hosts are registered with Activation Keys. Also, LE and CV, assigned to a host, can be changed independently of the AKs. So I'm not sure if we should mention AKs here.

**Comment by maximiliankolb** [comment]
> Hey @maximiliankolb, ultimately the package profile depends on the repos accessible to the host. So I guess mentioning the repo instead of the subscriptions would be better? I'll check with the SME once again regarding this. With regards to Activation Keys, not all hosts are registered with Activation Keys. Also, LE and CV, assigned to a host, can be changed independently of the AKs. So I'm not sure if we should mention AKs here.

Looking at https://docs.theforeman.org/nightly/Managing_Content/index-katello.html#Basic_Content_Management_Workflow_content-management, it say "create an AK; register hosts to consume content".

The packages available to a content host depend on its activation key(s). The repositories depend on the AK, but you don't assign repos to a host, but rather AKs.

Only during provisioning of EL, you can install from "synced content" which depends on a CV and LE (which is why those two fields are present on the create host page.

**Comment by jeremylenz** [comment]
> The packages available to a content host depend on its activation key(s). 

This is only true at registration time. It's still entirely possible to register a host with
```
subscription-manager register --org Default_Organization
```

with a username and password and no activation key, and enable repositories later.  It's also possible to register a host with an AK, and then change the repositories it has access to later.

Long story short: Activation keys only matter at registration time.

> The repositories depend on the AK, but you don't assign repos to a host, only AKs.

This is true at registration time. However, you can enable and disable repository sets on a host to your heart's content after registration.

File: guides/common/modules/proc_installing-packages-on-a-host.adoc (hunk 4 above)
Comment (jeremylenz) [reply guides/common/modules/proc_installing-packages-on-a-host.adoc:5]: It's more accurate to say it depends on the host's content view and lifecycle environment.

Activation keys only affect the host at registration time. CV and LCE can change at any time after registration, therefore affecting the packages available for installation.

**Comment by jeremylenz** [comment]
> Is this recommended or real-world usage? I have strong objects to recommend anything else than content management as documented.

It's not the most common workflow, but it's certainly supported. I'm not suggesting changing any recommendations; just being technically accurate in the documentation. The repos available depend on CV & LCE, not AK.

**Comment by AkshayGadhaveRH** [comment]
Hi @maximiliankolb, I have rebased this branch, is this good to merge?

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 7]
```diff
@@ -0,0 +1,15 @@
+[id="Installing_Packages_on_a_Host_{context}"]
+= Installing Packages on a Host
+
+Use this procedure to review and install packages on a host using the {ProjectWebUI}.
+The list of packages available for installation depends on the Content View and Lifecycle Environment assigned to the host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select the host you want to install packages on.
+. On the *Content* tab, click *Packages* > click the vertical ellipsis icon next to the upgrade button, and select *Install Packages*.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:9]:
```suggestion
. On the *Content* tab, select the *Packages* tab.
. On the vertical ellipsis icon next to the upgrade button, click *Install Packages*.
```

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+. A window showing a list of packages available for installation on the host will appear.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:10]:
```suggestion
```

File: guides/common/modules/proc_installing-packages-on-a-host.adoc [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+. Select the package or packages that you want to install on the host.
```
Comment (maximiliankolb) [inline guides/common/modules/proc_installing-packages-on-a-host.adoc:11]:
```suggestion
. In the *Install packages* window, select the package or packages that you want to install on the host.
```

**Review by maximiliankolb — COMMENTED** [review]
I made some **suggestions** on how to _find_ the "install packges" features.

What do you think?

File: guides/common/modules/proc_installing-packages-on-a-host.adoc (hunk 7 above)
Comment (AkshayGadhaveRH) [reply guides/common/modules/proc_installing-packages-on-a-host.adoc:9]: This definitely looks way better! Thanks!

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   9e406e5fe..875974b50  3.4 -> 3.4
   40588715d..1063a3b09  3.3 -> 3.3
   3cee1901e..a5612fa64  3.2 -> 3.2
